package.json

```diff
@@ -171,6 +171,18 @@
     "packages/connectivity-tests"
   ],
   "overrides": {
-    "cookie": "^0.7.2"
+    "cookie": "^0.7.2",
+    "mongodb-connection-string-url": "^7.0.0",
+    "kerberos": "^7.0.0",
+    "mongodb-client-encryption": "^7.0.0",
+    "bson": "^7.0.0",
+    "mongodb": "^7.0.0"
+  },
+  "dependencies": {
+    "mongodb-connection-string-url": "^7.0.0",
+    "kerberos": "^7.0.0",
+    "mongodb-client-encryption": "^7.0.0",
+    "bson": "^7.0.0",
+    "mongodb": "^7.0.0"
```

Comment:
^ These additions are intended to be temporary as a way of making sure the package-lock file doesn't start hoisting other versions of these dependencies, they'll need to be removed before merging